distributed/worker.py

```diff
@@ -303,7 +303,12 @@ class Worker(BaseWorker, ServerNode):
     scheduler_file: str, optional
     host: str, optional
     data: MutableMapping, type, None
-        The object to use for storage, builds a disk-backed LRU dict by default
+        The object to use for storage, builds a disk-backed LRU dict by default.
+
+        If a callable to construct the storage object is provided, it
+        will receive the worker's attr:``local_directory`` as an
+        argument if the calling signature has an argument named
+        ``worker_local_directory``.
```

Comment:
```suggestion
        If data is a callable and it has the argument ``worker_local_directory`` in its
        signature, it will be filled with the worker's attr:``local_directory``.
```

Reply:
Done but `s/it has/has`